Edge/Controllers/SettingsController.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using Edge.Database.Models;
+using Edge.DTO;
+using Edge.Utils.Conversion;
+
+namespace Edge.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class SettingsController : ControllerBase
+    {
+        private readonly Context context;
+
+        public SettingsController(Context context)
+        {
+            this.context = context;
+        }
+
+        [HttpGet]
+        public IActionResult Get()
+        {
+            Settings settings = context.Settings.First();

```

Comment:
We only access the first because there is only one home associated with an edge, right ?